doc/design/dist/submit-job.md

```diff
@@ -0,0 +1,137 @@
+
+# PaddlePaddle Client
+PaddlePaddle client is command line tool, you can use a PaddlePaddle client to start a local train and submit a distributed training job to kubernetes cluster.
+
+The relation of PaddlePaddle, kubernetes and docker:
+
+<img src="./submit-job.png" width="500">
+
+
+# Running your training locally
+Execute `paddle local train` to run your local train.
+```bash
+paddle local train
```

Comment:
The first argument after `paddle` should be a command. `local` isn't even a verb. It seems that it could simply be  `paddle train --locally` or just `paddle train` without Kubernetes related arguments.